apps/yara/CMakeLists.txt

```diff
@@ -14,22 +14,14 @@ message (STATUS "Configuring apps/yara")
 # Dependencies
 # ----------------------------------------------------------------------------
 
-# Disable older MSVC.
-if (MSVC AND MSVC_VERSION LESS 1800)
-    message (STATUS "  Visual Studio 2012 or newer required to build yara.")
-    return ()
-endif ()
-
-# Disable parallelism in MSVC as it supports only OpenMP 2.0.
-if (NOT MSVC)
```

Comment:
> Okay seriously, I don't like this solution either, so you can disable MSVC for now.

Will do.

> Where and how do you do find_package now? Are OpenMP flags added to all targets?

Its in this diff, so feel free to do some easter egg searching ;-) Actually for BUILD_SYSTEM==DEVELOP we detect the dependencies once in `build_system_init()` and then add them to every target. For release builds everything is still done locally.

> Why don't you make Windows/ICC builds a priority? Windows/MSVC builds suck because of OpenMP 2.0...

Let me think... I think it's because I am neither responsible for Intel stuff nor for Windows stuff :bulb:  (although I have done 90% of making the library work with ICC).

I think @rrahn, @marehr and @cpockrandt  would all be more suited for the task. @cpockrandt and I tried to make "Clang with Microsoft Codegen"/C2 work to improve the situation on Windows, but cmake on Windows doesn't work with it, yet. Also we haven't even been given access to the Windows version of ICC, just Linux right now (I will discuss that with Intel folks here at IXPUG).

But I agree, we really should get an alternative to MSVC!